modules/chassis_elasticsearch/manifests/init.pp

```diff
@@ -102,5 +102,19 @@
       ],
       before  => Chassis::Wp[ $config['hosts'][0] ],
     }
+
+    # Create shared config directory and give write permissions to web server.
+    $package_symlinks = $options[instances].map |$index, $value| { "/etc/elasticsearch/${value}/config" }
+    file { '/usr/share/elasticsearch/config':
+      ensure  => directory,
+      owner => 'elasticsearch',
+      group => 'www-data',
+      mode => '0777',
+      require => Elasticsearch::Instance[ $options[instances] ],
+    } ~>
```

Comment:
Can we use `notify` instead here? It's clearer than the operator which is easy to miss. (Actually, seems like this should be a dependency rather than a notification anyway?)